client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -6,6 +6,7 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var cartValues = require( 'lib/cart-values' ),
+	support = require( 'lib/support' ),
```

Comment:
I'm not sure `support` is the best name for this collection of links to our support pages. It's too ambiguous, IMHO. The fact that `support` can also be interpreted as a verb doesn't help ;/ Maybe something like `supportLinks` (urls?) or just `links`?